app/src/main/java/org/mozilla/focus/settings/privacy/PrivacySecuritySettingsFragment.kt

```diff
@@ -31,20 +31,23 @@ class PrivacySecuritySettingsFragment :
     override fun onCreatePreferences(p0: Bundle?, p1: String?) {
         addPreferencesFromResource(R.xml.privacy_security_settings)
 
-        val biometricPreference: SwitchPreferenceCompat? = findPreference(getString(R.string.pref_key_biometric))
+        val biometricPreference: SwitchPreferenceCompat? =
+            findPreference(getString(R.string.pref_key_biometric))
         val appName = getString(R.string.app_name)
         biometricPreference?.summary =
             getString(R.string.preference_security_biometric_summary2, appName)
 
         // Remove the biometric toggle if the software or hardware do not support it
         if (Build.VERSION.SDK_INT < Build.VERSION_CODES.M || !requireContext().canUseBiometricFeature()
         ) {
-            preferenceScreen.removePreference(biometricPreference)
+            biometricPreference?.let { preferenceScreen.removePreference(it) }
         }
         if (!FocusNimbus.features.onboarding.value().isCfrEnabled ||
             !requireContext().settings.shouldShowPrivacySecuritySettingsToolTip
         ) {
-            preferenceScreen.removePreference(findPreference(getString(R.string.pref_key_tool_tip)))
+            val privacySecuritySettingsToolTip: PreferenceToolTipCompose? =
+                findPreference(getString(R.string.pref_key_tool_tip))
+            privacySecuritySettingsToolTip?.let { preferenceScreen.removePreference(it) }
```

Comment:
The reason is in the commit message (I can add comments if it is not visible enough): 

> PreferenceGroup.removePreference now expects a NonNull preference as a param.

And `findPreference` returns a nullable preference. I also used a variable, because I found it easier to read than the one-liner with let in the middle:

Reply:
Ah, thanks - I was breezing through the review and didn't go through it as individual commits. 🙂